### ISSUE TITLE
average srcc on individual distoration type

### ISSUE THREAD

**Issue by yl-precious** [zwx8981/DBCNN-PyTorch#3]
when I uesed Gblur part in the LIVE database to train the network,  I got only 0.3 srcc at most on its 20% test. I guess the reason is lack of data, so I split the picture into 100*100 patch, also get low srcc, could you give me some advice~, thank you

**Comment by zwx8981** [comment]
@yl-precious Did you train the network from scratch? You can try our scnn, which is pre-trained using large amount of synthesized data. Take care of your learning rate and batchsize,  which may significant affect the result. In our implementation, setting learning rate to be 1e-5 and batchsize of 8 with Adam optimizer is good.

**Comment by yl-precious** [comment]
~~thanks for your reply. I didn't train it form scratch. to be exactly,I set learning rate to 1e-3 to train fc-layer with SGD optimizer, and then I set learning rate to 1e-5 to finetune the whole DBCNN with Adam optimizer as mentioned in your paper.  In  the training process,  I only uesed 174 images in LIVE's Gblur part.  When I used all the LIVE database to finetune, it works very well.  The individual distortion types results in your paper are directly obtained on the well-trained DBCNN? or using single distortion type images to finetune the network...

**Comment by zwx8981** [comment]
@yl-precious For testing performance of individual distortion types, we did not re-train the network using the images associated with the distortion type to be tested. The common practice adopted in IQA papers is directly testing the performance of the distortion type using the model trained with all images.

**Comment by yl-precious** [comment]
oh~, got it. another question is should I normalize the dmos to 0~1 or not?  could it influence the performence of the network?

**Comment by zwx8981** [comment]
@yl-precious  In our original implementation, we normalize the dmos or mos of different datasets to 0~100 , so I think it would be fine to normalize the dmos to 0~1 too.

**Comment by zwx8981** [comment]
@yl-precious I think it would be fine.

**Comment by yl-precious** [comment]
^=^ thank you
